homeassistant/components/spotify/strings.json

```diff
@@ -10,6 +10,7 @@
     "abort": {
       "already_setup": "You can only configure one Spotify account.",
       "authorize_url_timeout": "Timeout generating authorize url.",
+      "no_url_available": "No URL available",
```

Comment:
Also add this string to our common strings:
https://github.com/home-assistant/core/blob/8185ddf9a122228bd339df46ffe69aa449dc7f55/homeassistant/strings.json#L50-L56